Tests/Carbon_Group.Tests.ps1

```diff
@@ -143,13 +188,46 @@ Describe 'Carbon_Group' {
         $group.Members.Count | Should -Be 0
 
         # Change members
-        Set-TargetResource -Name $script:groupName -Members $script:username1 -Ensure 'Present'
+        ## EnsureMembers='Exact'
+        foreach( $i in 0..1 )

```

Comment:
I've added these to validate that `Set-TargetResource` does not fail, when there are no membership changes required (i.e. the second run).